client/src/components/body/EducatorResources/EducatorResources.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import WhatsAvailable from "./WhatsAvailable";
+import ResourcesHeader from "./ResourcesHeader";
+import ResourcesSection from "./ResourcesSection";
+
+const EducatorResources = () => {
+  return (
+    <div className={"tw-w-full"}>
+      <ResourcesHeader />
```

Comment:
Should these sections be taking up the whole page?

Seems odd that they're not.
![image](https://github.com/user-attachments/assets/54fa9802-261b-4d23-9b15-9e48dfefca12)

Reply:
This is how it looks like in the figma. Seems like it is taking up the whole page, with the picture being at the bottom
![image](https://github.com/user-attachments/assets/4a7f5d4a-84b7-4d1b-8b4e-9e3d6ac4e20c)

Reply:
done!